packages/ui/src/pages/Chat/chat.tsx

```diff
@@ -36,10 +36,14 @@ import { AuthService } from '@etherealengine/client-core/src/user/services/AuthS
 import '@etherealengine/engine/src/EngineModule'
 
 import { LocationService } from '@etherealengine/client-core/src/social/services/LocationService'
+import { clientContextParams } from '@etherealengine/client-core/src/util/contextParams'
+import multiLogger from '@etherealengine/common/src/logger'
 import { getMutableState } from '@etherealengine/hyperflux'
 import { NetworkState } from '@etherealengine/network'
 import { loadEngineInjection } from '@etherealengine/projects/loadEngineInjection'
 
+const logger = multiLogger.child({ component: 'engine:ecs:ProfileMenu', modifier: clientContextParams })
```

Comment:
I dont think `engine:ecs:ProfileMenu` is the correct value for `component` prop here.
```suggestion
const logger = multiLogger.child({ component: 'ui:chat:chat', modifier: clientContextParams })
```